.github/workflows/unit-test.yml

```diff
@@ -15,10 +15,12 @@ jobs:
             lerna-extra-args: >-
               --ignore @opentelemetry/instrumentation-aws-sdk
               --ignore @opentelemetry/instrumentation-pino
+              --ignore @opentelemetry/instrumentation-mongodb
               --ignore @opentelemetry/instrumentation-tedious
           - node: "10"
             lerna-extra-args: >-
               --ignore @opentelemetry/instrumentation-pino
+              --ignore @opentelemetry/instrumentation-mongodb
     runs-on: ubuntu-latest
```

Comment:
since we now run tests for version 3 in CI, we can keep running it on both node 8 and 10

Reply:
Fixed :)